sips/sip-x.md

```diff
@@ -0,0 +1,67 @@
+---
+sip: <to be assigned>
```

Comment:
could you pls number it 127 and rename the file? 🙏🏼

Reply:
Done in f3621d9